<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeCal</class>
 <widget class="QDialog" name="homeCal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="backFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>640</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="topBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
width: 1024px;
height: 100px;
left: 0px;
top: 0px;
text-align: center;

background: #FEFAE0;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="app_icon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 85px;
height: 85px;
image: url(D:/NotaProject/pics/notaKung.png);
left: 3px;
top: 5px;

background: #86B18B;

border-radius: 35px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>521</width>
       <height>53</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="welcomeUser">
        <property name="font">
         <font>
          <family>Tenor Sans</family>
          <pointsize>1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">position: absolute;
width: 190px;
height: 44px;
left: 103px;
top: 28px;
text-align: center;
font-family: Tenor Sans;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 21px;

color: #000000;</string>
        </property>
        <property name="text">
         <string>Welcome,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="date">
        <property name="styleSheet">
         <string notr="true">position: absolute;
width: 190px;
height: 44px;
left: 103px;
top: 28px;

font-family: Tenor Sans;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 21px;

color: #000000;</string>
        </property>
        <property name="text">
         <string>November 21, 2021</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>941</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
width: 1024px;
height: 640px;
left: 0px;
top: 0px;

background: #84B289;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>480</y>
       <width>209</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="cancelAdding">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
background: #EC8E65;
border-radius: 15px;</string>
        </property>
        <property name="text">
         <string>CANCEL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveNoteButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
background: #C6D57E;
border-radius: 15px;</string>
        </property>
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>821</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #E4E2C7;


</string>
     </property>
     <widget class="QTextEdit" name="taskName_textEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>759</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0,0,0,0);
border-radius: 10px;
background-color: rgb(244, 255, 228);
color: rgb(88, 105, 82);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string>TASKNAME</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="task_description">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>731</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0,0,0,0);
border-radius: 10px;
background-color: rgb(244, 255, 228);
color: rgb(88, 105, 82);</string>
      </property>
      <property name="placeholderText">
       <string>DESCRIPTION</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>721</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0,0,0,0);
border-radius: 10px;
background-color: rgb(244, 255, 228);
color: rgb(88, 105, 82);</string>
      </property>
      <property name="date">
       <date>
        <year>2021</year>
        <month>12</month>
        <day>6</day>
       </date>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="warning">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>500</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Please fill in task name!!</string>
     </property>
    </widget>
    <zorder>widget</zorder>
    <zorder>layoutWidget_2</zorder>
    <zorder>warning</zorder>
   </widget>
   <widget class="QFrame" name="leftBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>91</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
text-align: center;
position: absolute;
width: 140px;
height: 540px;
left: 0px;
top: 100px;

background: #A3C5A7;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="homeButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">position: absolute;

image: url(D:/NotaProject/pics/home.png);
width: 120px;
height: 37px;
left: 7px;
top: 16px;
text-align: right;

background: #C4C4C4;
border: 1px solid #000000;

border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="home_label">
           <property name="styleSheet">
            <string notr="true">text-align: center;</string>
           </property>
           <property name="text">
            <string>     HOME</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="timeTableButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(D:/NotaProject/pics/timetable.png);
position: absolute;
width: 120px;
height: 37px;
left: 7px;
top: 160px;
text-align: right;
background: #C4C4C4;
border: 1px solid #000000;

border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeTable_label">
           <property name="text">
            <string> TIMETABLE</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="noteButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(D:/NotaProject/pics/writing.png);
position: absolute;
width: 120px;
height: 37px;
left: 7px;
top: 160px;
text-align: right;
background: #C4C4C4;
border: 1px solid #000000;

border-radius: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="note_label">
           <property name="text">
            <string>    NOTE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="taskButton">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(D:/NotaProject/pics/task.png);
position: absolute;
width: 120px;
height: 37px;
left: 7px;
top: 160px;
text-align: right;
background: #ECF39E;


border-radius: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="task_label">
           <property name="text">
            <string>     TASK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="unAddTask">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>67</width>
          <height>61</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(0,0,0,0);
image: url(D:/NotaProject/pics/unAdd.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>frame_5</zorder>
   <zorder>topBar</zorder>
   <zorder>leftBar</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
